### ISSUE TITLE
workbench-voice option to choose a voice

### ISSUE THREAD

**Issue by valler** [microsoft/vscode#239442]
`accessibility.voice` should have an option to choose the voice for speech synthesis. For example, one should be able to pick Daniel (one of Siri's voices).

**Comment by meganrogge** [comment]
What are you using this for?

**Comment by meganrogge** [comment]
I assume you're using the `VS Code Speech` extension?

**Comment by valler** [comment]
Yes I have VS Code Speech installed.
